08.Packages.md

```diff
@@ -0,0 +1,12 @@
+# Packages
+
+Some contributors are creating and maintaining external libraries that could simplify your life on your project. Those contributors created packages that use Gitter API in different shapes (progamming languages, main goals, ...). You can retrieve a list of the current existing packages below, sorted by languages.
+
+## .NET
```

Comment:
There is no mention of C# so a `Ctrl + F` wouldn't find anything atm. Is it safe to say this will most likely get used in a C# environment?